tests/test_microstructure.py

```
@@ -0,0 +1,198 @@
+import numpy as np
+from pytest import mark, skip
```

Ruff (0.15.10)

[warning] Incorrect import of `pytest`; use `import pytest` instead

(PT013)